_docs/installing/openbsd.md

```diff
@@ -0,0 +1,32 @@
+---
+title: OpenBSD
+category: Installing the bot
+order: 8
+---
+
+MusicBot can run on OpenBSD systems as well. Note that the X11 sets must be installed, due to the `ffmpeg` dependency.
+
+## OpenBSD 6.6
+
+~~~ bash
+# Install Python and libraries available as packages
+doas pkg_add python # select version 3.7.4
+doas pkg_add py3-aiohttp youtube-dl ffmpeg libsodium git
+
+# Ensure pip is set up
+python3 -m ensurepip
+
+# Clone the MusicBot
+git clone https://github.com/Just-Some-Bots/MusicBot.git -b master
+cd MusicBot
+
+# Install remaining dependencies
+doas pip3 install -U pip
+doas pip3 install -U -r requirements.txt
+~~~
+
+WARNING: If you have the py3-PyNaCl package installed, the final command will overwrite your system pynacl, which is likely newer, with pynacl 1.2.1, potentially breaking other packages. This can be worked around either by using a virtualenv (safe), or editing requirements.txt to remove the pinned version (at your own risk).
```

Comment:
Could you move this warning above the command?
That way we can ensure people will read it instead of pasting the command and then reading.